package.json

```diff
@@ -9,7 +9,7 @@
     "node": ">=12.13"
   },
   "scripts": {
-    "docs": "yarn --cwd packages/docs start",
+    "docs": "yarn --cwd packages/docs start --no-clear-screen",
```

Comment:
nit: Move the `--no-clear-screen` flag into the `start` command inside of `packages/docs/package.json`

Reply:
Done